### PR TITLE
Documentation setup

### PR DESCRIPTION
**Pull request by liamegan**
This is the first step in the documentation setup. Take a look through the code provided, I've also created a new action that will hopefully build this and deploy as appropriate.

### CODE REVIEW

**Comment by liamegan** [comment]
resolves #25

File: documentation/components/navigation/_navigation.pug [hunk 1]
```diff
@@ -0,0 +1,14 @@
+mixin makeNavUL(items)
+  ul.nav
+    each val, index in items 
+      li
+        - if (typeof(val) === 'array') {
```
Comment (marlonmarcello) [inline documentation/components/navigation/_navigation.pug]: you don't need the dash `-` and javascript for conditionals, [pug has conditionals by default](https://pugjs.org/language/conditionals.html)

File: documentation/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "start": "sweet-potato-cooker start",
+    "build": "sweet-potato-cooker build",
+    "clean": "sweet-potato-cooker clean",
+    "compress": "sweet-potato-cooker compress"
```
Comment (marlonmarcello) [inline documentation/package.json]: Need to remove this. We don't compress with the cooker anymore. That should also be updated on the peeler templates.

File: documentation/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import BarfyStars from "https://cdn.skypack.dev/wtc-barfystars";
+
+const barfTitle = new BarfyStars(document.querySelector("h1"));
```
Comment (marlonmarcello) [inline documentation/pages/index.js]: I love this. Barfing titles. But I think it's leftovers from the peeler and I am just wondering if you indeed wanna keep it.

**Review by marlonmarcello — CHANGES_REQUESTED** [review]

File: documentation/package.json (hunk 2 above)
Comment (liamegan) [reply documentation/package.json]: Ah yeah that's straight off the minimal template. I'll update that in another PR

File: documentation/pages/index.js (hunk 3 above)
Comment (liamegan) [reply documentation/pages/index.js]: Haha yeah it is.

File: documentation/components/navigation/_navigation.pug (hunk 1 above)
Comment (liamegan) [reply documentation/components/navigation/_navigation.pug]: Yeah my bad there. I clearly haven't spent much time with Pug recently.

File: documentation/components/navigation/_navigation.pug [hunk 4]
```diff
@@ -0,0 +1,13 @@
+mixin makeNavUL(items)
+  ul.nav
+    each val, index in items 
+      li
+        if (typeof(val) === 'array')
```
Comment (marlonmarcello) [inline documentation/components/navigation/_navigation.pug]: Arrays are considered `objects`. You have to do `val instanceof Array` if you wanna make sure it's an array

**Review by marlonmarcello — CHANGES_REQUESTED** [review]

File: documentation/components/navigation/_navigation.pug (hunk 4 above)
Comment (liamegan) [reply documentation/components/navigation/_navigation.pug]: Well that's embarrassing

**Review by marlonmarcello — APPROVED** [review]

**Pull request merged**